bin.js

```diff
@@ -271,11 +271,9 @@ function build (entry, outputDir, argv, done) {
     if (argv.uglify !== false && argv.uglify !== 'false') {
       log.debug('uglify starting')
       js = uglify.minify(js, {
-        fromString: true,
-        compress: true,
-        mangle: true,
-        filename: file,
-        sourceMaps: false
+        sourceMap: {
+          filename: file
```

Comment:
yeah those are the options, and do we need the filename? - just gotta make sure we're not including source maps unless through `--debug` haha :sparkles: